examples/src/Viewer.tsx

```diff
@@ -507,13 +471,18 @@ class Viewer extends React.Component<InputParams, ViewerState> {
             []
         );
         const uniqueTags: string[] = [...new Set(allTags)];
-        if (isEqual(uiDisplayData, this.state.selectionStateInfo.appliedColors)) {
+        if (
+            isEqual(uiDisplayData, this.state.selectionStateInfo.appliedColors)
+        ) {
```

Comment:
idk if anyone's especially committed to it, but hot take this repo's 80 column limit is a bit too small

Reply:
Sizzler! I like it though... @meganrm what do you think about bumping the prettier printWidth to 120? We will vibe with volume-viewer then.

AICS style guide anyone? Would actually be nice if this sort of thing was consistent across repos.

Reply:
that's fine with me